bpf/lib/nodeport.h

```diff
@@ -911,8 +987,28 @@ static __always_inline int nodeport_lb6(struct __ctx_buff *ctx,
 #endif
 		ctx_set_xfer(ctx, XFER_PKT_NO_SVC);
 
-		if (nodeport_uses_dsr6(&tuple))
+#ifdef ENABLE_DSR
+		if (nodeport_uses_dsr6(&tuple)) {
+			bool dsr = false;
+
+			ret = nodeport_extract_dsr_v6(ctx, ip6, &key.address,
+						      &key.dport, &dsr);
```

Comment:
Do we even need `nodeport_extract_dsr_v6` now? It looks like `find_dsr_v6` could write directly into `key.address` and `key.dport`.

Reply:
If you look at the final result of the PR, there's quite a bit of surrounding logic in `nodeport_extract_dsr_v6()` that deals with TCP-SYN etc.

Reply:
Oh, right!